docker/importer/volume/surveillance-mediator/endpoint-resource-resolver-dhis.json

```
@@ -0,0 +1,100 @@
+{
+  "name": "Covid19 DHIS2 Orchestrator",
+  "description": "Endpoint for mapping the Covid19 case report into DHIS format and then forwarding it to DHIS2",
+  "endpoint": {
+    "pattern": "/resource-resolver-dhis"
+  },
+  "transformation": {
+    "input": "JSON",
+    "output": "JSON"
+  },
+  "inputValidation": {
+    "type": "object",
+    "properties": {
+      "requestBody": {
+        "type": "object"
```

A bit more validation would have highlighted the data issue here

```suggestion
        "type": "object",
        "properties": {
          "data": {
            "type": "object",
            "properties": {
              "identifier": {
                "type": "string"
              },
              "observations": {
                "type": "array"
              }
            },
            "required": ["identifier", "observations"]
          }
        },
        "required": ["data"]
```